app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/snippet/SnippetDetailsViewModel.kt

```diff
@@ -0,0 +1,184 @@
+package com.bottlerocketstudios.brarchitecture.ui.snippet
+
+import com.bottlerocketstudios.brarchitecture.R
+import com.bottlerocketstudios.brarchitecture.data.converter.convertToComment
+import com.bottlerocketstudios.brarchitecture.data.converter.convertToUiModel
+import com.bottlerocketstudios.brarchitecture.data.converter.convertToUser
+import com.bottlerocketstudios.brarchitecture.data.repository.BitbucketRepository
+import com.bottlerocketstudios.brarchitecture.domain.models.SnippetComment
+import com.bottlerocketstudios.brarchitecture.domain.models.SnippetDetailsFile
+import com.bottlerocketstudios.brarchitecture.domain.models.Status
+import com.bottlerocketstudios.brarchitecture.domain.models.User
+import com.bottlerocketstudios.brarchitecture.ui.BaseViewModel
+import com.bottlerocketstudios.compose.snippets.SnippetUiModel
+import com.bottlerocketstudios.compose.snippets.snippetDetails.SnippetDetailsUiModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import org.koin.core.component.inject
+
+class SnippetDetailsViewModel : BaseViewModel() {
+    // DI
+    private val repo: BitbucketRepository by inject()
+
+    // State
```

Comment:
Let's rename these "State" comments to "UI".  I usually reserve "State" for internal state that is not connected to UI.